package/src/components/FinalReviewCheckoutAction/v1/FinalReviewCheckoutAction.js

```diff
@@ -139,7 +139,8 @@ class FinalReviewCheckoutAction extends Component {
         CartSummary
       },
       label,
-      stepNumber
+      stepNumber,
+      ...props
```

Comment:
@nnnnat Should I add `productURLPath` explicitly as a prop to `FinalReviewCheckoutAction`, or is passing it through implicitly though `...props` okay?